### PR TITLE
Add Partnership Page

### LINKED ISSUE #49: Add partnership info page
https://www.figma.com/design/Z1Ol2UJNsUbdCc2lrCUoWF/WEBSITE-MASTER?node-id=44-262&node-type=frame&t=1jPvVtr1lTIh4WH2-0

### PR DESCRIPTION
**Pull request by clementinelove**
Closes #49

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const navigation: NamedRoute[] = [
   { text: "Home", to: "/" },
   { text: "Causes", to: "/causes" },
   { text: "About Us", to: "/about" },
+  { text: "Partnership", to: "/charity-partnership" },
```
Comment (JElgar) [inline src/components/Navbar.tsx:20]: Is there a reason you went with `charity-partnership` over just `partnership`?

**Review by JElgar — APPROVED** [review]
Looks awesome thank you!

File: src/components/Navbar.tsx (hunk 1 above)
Comment (clementinelove) [reply src/components/Navbar.tsx:20]: Um this is the one I was told to use:
> (https://docs.google.com/document/d/1EX9VHtpoX5KbRTffs5GV5drv70t1oIei/edit#heading=h.sv7xkgqj8ksr)
> - [Partnership figma](https://www.figma.com/design/Z1Ol2UJNsUbdCc2lrCUoWF/WEBSITE-MASTER?node-id=44-262&node-type=frame&t=1jPvVtr1lTIh4WH2-0)
> - Page name: Charity Partnerships
> - Slug: charity-partnership/
> - URL: https://www.now-u.com/charity-partnership/
> - Partner with us CTA link: https://forms.monday.com/forms/cc4aaf4815a82d3f5450e5f519a12f1f?r=euc1

_Originally posted by @YWKlai in https://github.com/now-u/now-u-web/issues/49#issuecomment-2415556984_

Comment (JElgar) [reply src/components/Navbar.tsx:20]: Ok perfecto. Looks strange to me but those folks know much better than I do

**Pull request merged**